src/main/java/in/twizmwaz/cardinal/util/Players.java

```diff
@@ -42,6 +43,8 @@ public static String getName(ServerOperator who, boolean flairs) {
             if (player.isOnline()) {
                 if (flairs) {
                     return player.getPlayer().getDisplayName();
+                } else if (TitleRespawn.isDeadUUID(player.getUniqueId())) {
+                    return ChatColor.DARK_GRAY + "" + ChatColor.BOLD + player.getPlayer().getDisplayName();
```

Comment:
name in list should change to gray, but if they writte stuff on the chat it should be their regular name (team colored), you should probably only make a "setListName" or something like that, that sets their name to their flags + gray, and when they respawn get it back to what it was